sonic_platform_base/device_base.py

```diff
@@ -11,6 +11,17 @@ class DeviceBase(object):
     peripheral device
     """
 
+    #Device-types
+    DEVICE_TYPE_PSU     = "PSU"
+    DEVICE_TYPE_FAN     = "FAN"
+    DEVICE_TYPE_FANDRAWER = "FAN-DRAWER"
```

Comment:
Why do we need these in DeviceBase? DeviceBase is meant to be an abstract base class with no concept of the types of classes which inherit from it.

Reply:
Removed it since the implementation in PSUd has changed. Previously, we were getting a list of power consumers and wanted a common place for standard device-types. Also, modules are now limited to supervisor, line-cards and fabric-cards.